### PR TITLE
merge cli with api

### PR DESCRIPTION
**Pull request by drgarcia1986**
To avoid dealing with two projects we can put teresa-cli inside of teresa-api

---
This change is [Reviewable](https://reviewable.io/reviews/luizalabs/teresa-api/176)

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,11 +2,15 @@
 All notable changes to this project will be documented in this file.
 This project adheres to [Semantic Versioning](http://semver.org/).
 
+## [NEXT_RELEASE]
+### Added Changed
+- Merge `teresa-cli` inside of `teresa-api` repo
+
 ## [0.3.0] - 2017-04-07
 ### Added
```
Comment (aguerra) [inline CHANGELOG.md]: I guess merge `teresa-cli` into (or with)  `teresa-api` repo better.

File: Makefile [hunk 2]
```diff
@@ -85,3 +85,6 @@ validate:
 
 swagger-docs:
 	go run docs/webserver.go swagger.yml
+
+build-client:
+	@go build -o teresa cmd/client/main.go
```
Comment (aguerra) [inline Makefile]: Does this go build command use the vendor dir automatically?

Comment (drgarcia1986) [inline Makefile]: I guess yes, but I'm going to make some tests.

Comment (drgarcia1986) [inline Makefile]: Works like a charm

**Comment by arnaldopereira** [comment]
*[CHANGELOG.md, line 10 at r1](https://reviewable.io:443/reviews/luizalabs/teresa-api/176#-Ki23MJn0iqM0YUs26ZL-r1-10:-Ki6rq3gP-NhHtSzcVZZ:b6v3trh) ([raw file](https://github.com/luizalabs/teresa-api/blob/390012243561ce7ec9395385fcad1365dead206f/CHANGELOG.md#L10)):*
<details><summary><i>Previously, aguerra (Alex Guerra) wrote…</i></summary><blockquote>

I guess merge `teresa-cli` into (or with)  `teresa-api` repo better.
</blockquote></details>

I'd use `merge teresa-cli and teresa-api`

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/luizalabs/teresa-api/176)*

**Comment by arnaldopereira** [comment]
Reviewed 299 of 300 files at r1, 1 of 1 files at r2.
Review status: all files reviewed at latest revision, 2 unresolved discussions.

---



*Comments from [Reviewable](https://reviewable.io:443/reviews/luizalabs/teresa-api/176)*

**Comment by arnaldopereira** [comment]
*[CHANGELOG.md, line 10 at r1](https://reviewable.io:443/reviews/luizalabs/teresa-api/176#-Ki23MJn0iqM0YUs26ZL-r1-10:-KiW1YVQrnP4fYQaCNUs:bb6ywmx) ([raw file](https://github.com/luizalabs/teresa-api/blob/390012243561ce7ec9395385fcad1365dead206f/CHANGELOG.md#L10)):*
<details><summary><i>Previously, arnaldopereira (Arnaldo de Moraes Pereira) wrote…</i></summary><blockquote>

I'd use `merge teresa-cli and teresa-api`
</blockquote></details>

LGTM

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/luizalabs/teresa-api/176)*

**Comment by arnaldopereira** [comment]
Reviewed 1 of 1 files at r3.
Review status: all files reviewed at latest revision, 2 unresolved discussions.

---



*Comments from [Reviewable](https://reviewable.io:443/reviews/luizalabs/teresa-api/176)*

**Pull request merged**